packages/react-sdk/src/lib/text-formatting/useFontFamily.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2025 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useCallback } from 'react';
+import {
+  useActiveElements,
+  useElements,
+  useWhiteboardSlideInstance,
+} from '../../state';
+import { TextFontFamily } from '../../state/crdt/documents/elements';
+import { calculateFontFamilyUpdates } from './calculateFontFamilyUpdates';
+
+type UseFontSizeResult = {
+  fontFamily?: TextFontFamily;
+  setFontFamily: (value?: TextFontFamily) => void;
+};
+
+export function useFontFamily(): UseFontSizeResult {
+  const slideInstance = useWhiteboardSlideInstance();
+  const { activeElementIds } = useActiveElements();
+  const activeElements = useElements(activeElementIds);
+  const elements = Object.values(activeElements);
+
+  let fontFamily: TextFontFamily | undefined = 'inter';
+
+  for (const element of elements) {
+    if (element.type === 'shape') {
```

Comment:
Not sure, if we should use the same checks than here https://github.com/nordeck/matrix-neoboard/blob/624c21ffdf31f3074122e244e4d99b3632771653/packages/react-sdk/src/components/Whiteboard/WhiteboardHost.tsx#L70

Reply:
This would now apply to shapes with no text. Please see the link. We should search the first text shape and use the font from it.

Reply:
This is still not done. The font of the first shape with text should be used.